tensorboard/components/tf_tensorboard/tf-tensorboard.html

```diff
@@ -146,14 +145,6 @@ <h2>Settings</h2>
 
       <div id="content-pane" class="fit">
         <slot id="injected-overview"></slot>
-        <template is="dom-if" if="[[_isDataSelectorEnabled]]">
-          <div id="data-selector">
-            <tf-collapsible-data-selector
-              selection="{{_dataSelection}}"
-              active-plugins="[[_activeDashboards]]"
-            ></tf-collapsible-data-selector>
-          </div>
-        </template>
         <div id="content">
           <template is="dom-if" if="[[_activeDashboardsFailedToLoad]]">
             <div class="warning-message">
```

Comment:
GitHub doesn’t let me comment in the right place, but:

```
$ git grep -n data.selector
tensorboard/components/tf_tensorboard/tf-tensorboard.html:314:      #data-selector {
tensorboard/components/tf_tensorboard/tf-tensorboard.html:318:      tf-collapsible-data-selector {
tensorboard/components/tf_tensorboard/tf-tensorboard.html:319:        --tf-collapsible-data-selector: {
tensorboard/components/tf_tensorboard/tf-tensorboard.html:323:        --tf-collapsible-data-selector-data-selector: {
```

Reply:
Thanks, I totally forgot about the css parts